tests/unit/mock_interop.mof

```diff
@@ -0,0 +1,123 @@
+// This is a simple mof model that creates the qualifier declarations,
+// classes, and instances for a very simplistic model to be used in the
+// pywbemcli mock test environment.
```

Comment:
Lets add the following  to the comments. This creates a minimal interop namespace including the class for CIM_Namespace and CIMObjectManager so that commands like namespace list will confirm that the required interop namespace exists.

Reply:
DONE, but with different text:
```
// This is a simple mof model that creates the qualifier declarations,
// classes, and instances for a very simplistic model of the Interop namespace
// to be used in the pywbemcli mock test environment.
// The resulting Interop namespace includes the classes for CIM_Namespace and
// CIM_ObjectManager and an instance of CIM_ObjectManager so that the needs
// of the WBEM server functionality of pywbem are satisfied. The instances
// of CIM_Namespace (including the one for the Interop namespace itself)
// will be created by the namespace provider.
```